### DISCUSSION TITLE
How to customize search highlight?

### DISCUSSION THREAD

**Discussion by tommyalatalo** [sharkdp/bat#2480] [Q&A] [answered]
I just set up bat as my man page viewer with the hopes to improve the experience, but I have this issue where after searching with `/` the highlighting of the match is very poor (the matched text is only set as italic, which is incredibly difficult to spot in a full document).

I'm using the gruvbox-dark theme, so I'm not sure if that's where this should be changed. In any case I would like to get something like a text background color applied to the matches, that would make it so much easier to see.

Any help much appreciated

**Comment by keith-hall** [comment, marked as answer]
Hi, thanks for raising your question and sorry for the slow reply. I don't think this behavior of `less` is something `bat` can control. Maybe https://unix.stackexchange.com/q/179173/152147 can help?

**Reply by tommyalatalo** [reply]
Thanks for the response, I'll mark it as an answer, though what I ended up doing was setting up Neovim as my manpager, it fulfills my needs perfectly while still keeping bat as a replacement for cat. :)
